src/test/java/it/gov/pagopa/payment/controller/payment/BarCodePaymentControllerIntegrationTest.java

```diff
@@ -210,7 +221,7 @@ void authorizeTransaction() throws Exception{
         TransactionBarCodeResponse trxCreated = createTrxSuccess(trxRequest, USERID);
 
         // Trying to authorize the bar code with a merchant not onboarded on the initiative
-        extractResponse(authTrx(trxCreated.getTrxCode(), authBarCodePaymentDTO, "DUMMYMERCHANTID"), HttpStatus.FORBIDDEN, null);
+        extractResponse(authTrx(trxCreated.getTrxCode(), authBarCodePaymentDTO, "DUMMYMERCHANTID"), HttpStatus.NOT_FOUND, null);
```

Comment:
i'm not expecting that this task should change the previous responses! why should this change from FORBIDDEN to NOT_FOUND?